pendulum:
  id: "0000"  # The integer unique identifier used for data storage
  length: 1.0  # The pendulum length [m]
  axis_height: 1.0  # The pendulum axis height [m] (Not used in the simulation. Height center body + length)
  added_mass: 1.0  # The pendulum added mass (not including the pendulum mass itself) [kg]
  tip_type: "default"  # The pendulum tip type